app/custom/molecules/menu/_style.scss

```diff
@@ -0,0 +1,22 @@
+nav[role="navigation"] {
```

Comment:
I think we should use single quotes instead of double in order to follow the same lint rule on PHP, JS and CSS